lib/puppet/provider/package/chocolatey.rb

```diff
@@ -178,7 +178,7 @@ def self.instances
           else
             values = line.split(' ')
           end
-          packages << new({ :name => values[0], :ensure => values[1], :provider => self.name })
+          packages << new({ :name => values[0].downcase, :ensure => values[1], :provider => self.name })
```

Comment:
Hi @magmax - Your commit message doesn't say much, so I'm going to ask, why is this necessary? What version where you on before? 

Was it 0.5.0 / 0.5.1? https://github.com/chocolatey/puppet-chocolatey/commit/cb7f0e5e33ddef445af4a03b9ab339588a9dc698